### PR TITLE
Added windows support

### PR DESCRIPTION
**Pull request by Oro**
I've added installing the sensu client on windows (tested on win2012), but I am unsure if the split up is pythonic (or saltastic :) )
If you have any objections or better ways of handling e.g. file mode (which does not exist on windows but is necessary on linux) please tell me

### CODE REVIEW

**Comment by powellchristoph** [comment]
Thanks for the PR. I don't use Windows so I really have zero input on running it there, but how is it getting installed? For Linux support, we are adding the repository and using the OS package manager to install it. You have added logic to skip it if the OS is Windows.

File: sensu/init.sls [hunk 1]
```diff
@@ -7,6 +7,7 @@ python-apt:
 {% endif %}
 
 sensu:
```
Comment (powellchristoph) [inline sensu/init.sls]: Could this block be modified to be more like this pseudo-code:
```
sensu:
    if OS_familly == windows:
        download and install the msi package
    else
        configure/install with repos
```

**Comment by Oro** [comment]
For windows, it is using [The Windows Software Repository](https://github.com/saltstack/salt-winrepo) (I've added the sensu client there with a pull request), see also the documentation at http://docs.saltstack.com/en/latest/topics/windows/windows-package-manager.html
The repo location is set on the master (master config) and not on the different clients, which is why I needed to remove the pkgrepo state for windows minions.

**Comment by powellchristoph** [comment]
Ahh, like I said, zero windows input! Could you add a small section to the readme outlining windows support? I wasn't familiar with the salt-winrepo. Thanks again for the PR.

**Pull request merged**